app/models/journal_cutoff_date.rb

```diff
@@ -23,6 +23,33 @@ def past?
     cutoff_date < Time.current
   end
 
+  # Admin staff can still close journals for about a week after the end of the fiscal year.
+  # This is known as the 'year end closing window' - the date range between the
+  # end of the fiscal year and the journal cutoff date for the last month of the fiscal year.
+  #
+  # For example:
+  #   With FY22 starting 09/01/21, the closing window would
+  #   start Sep 1, 2021 and end on the August 2021 cutoff date:
+  #   Sep 1, 2021 - Sep 8, 2021.
+  #
+  # Returns true or false.
+  def self.year_end_closing_window?
```

Comment:
Is there a spec which shows whether the `now < year_end_cutoff.cutoff_date` is working as expected when the `Time.current` is on the same date as the cutoff_date.

Or when the Time.now is the same date as the `last_valid_date`?